std/zlib.d

```diff
@@ -149,7 +149,7 @@ unittest
  *     the compressed data
  */
 
-const(void)[] compress(const(void)[] srcbuf, int level)
+ubyte[] compress(const(void)[] srcbuf, int level)
```

Comment:
It seems off to take a `void[]` and return a `ubyte[]`. Both should be plain data, a.k.a. `ubyte[]`, e.g. it doesn't make sense to compress a `Object[]` casted to `void[]`.
As this would break the API you might want to add a `ubyte[]` overload or so.